### ISSUE TITLE
Incorrect results when running screen inside of iTerm

### ISSUE THREAD

**Issue by technicalpickles** [jamestalmage/supports-hyperlinks#3]
iTerm's [Proprietary Escape Code documentation](https://iterm2.com/documentation-escape-codes.html) says:

> These may not work properly in tmux or screen

This doesn't work as far as I can tell so far. If that is expected, I think there could be some additional checks in:

 https://github.com/jamestalmage/supports-hyperlinks/blob/49a52c1ad86eeb0970c908cc8ad57328554b16c2/index.js#L53-L59 

Specifically `process.env.TERM` seems to be `screen` for me, even when `TERM_APP` is iTerm.app

**Comment by jamestalmage** [comment]
If I'm understanding you correctly. It should be false, but you're getting true?

**Comment by technicalpickles** [comment]
That's right. It has been a bit, so I don't have the code handy to use it, but hyperlinks don't work inside of screen even if using iTerm.

**Comment by wodin** [comment]
I just ran into this yesterday.

Hyperlinks work in `iTerm2` and `supports-hyperlinks` correctly says they do:
<img width="467" alt="iTerm2 supports hyperlinks" src="https://user-images.githubusercontent.com/4579020/93708186-80714700-fb34-11ea-815b-32547e5e72ea.png">

Hyperlinks do not work in `tmux` running inside `iTerm2`, but `supports-hyperlinks` claims they do:
<img width="467" alt="tmux does not support hyperlinks" src="https://user-images.githubusercontent.com/4579020/93708193-97179e00-fb34-11ea-8633-e9da96e45f39.png">

**Comment by LitoMore** [comment]
The behavior is correct. You could use `FORCE_HYPERLINK` to test hyperlinks compatibility in `tmux` or `screen`.

```sh
npm i -g terminal-link-cli
FORCE_HYPERLINK=1 terminal-link text http://github.com
```

**Comment by wodin** [comment]
> The behavior is correct.

I've just tested it again now and it is indeed correct *now* for tmux.

When I ran into this issue, `supports-hyperlinks` was returning `true` for `supportsHyperlinks.stdout` and `supportsHyperlinks.stderr` when it was run under tmux.

I tried now with version 2.3.0 it returns `false` under tmux, as it should.

Unfortunately it still returns `true` when running under screen, despite screen not supporting hyperlinks.
